src/components/Header/Header.js

```diff
@@ -22,18 +21,19 @@ const Header = ({currentUser, logout}) => {
       <Navbar.Brand>
       <Logo />
       <Text b color="inherit" hideIn="xs">
-      Buddy
+        BUDDY
       </Text>
       </Navbar.Brand>
       <Navbar.Content hideIn="xs">
-          <Navbar.Link href="/">Home</Navbar.Link>
-          <Navbar.Link href="/Friends">Friends</Navbar.Link>
-          <Navbar.Link href="#">Search</Navbar.Link>
-          <Navbar.Link href="/BuddyProfile">BuddyProfile</Navbar.Link>
+        <NavLink to="/" >HOME</NavLink>
+        <NavLink to="/aboutus" >ABOUT US</NavLink>
+        <NavLink to={`/buddyprofile/${currentUser?.id}` }>BUDDY PROFILE</NavLink>
+
         </Navbar.Content>
+
       <Navbar.Content>
         <Navbar.Link color="inherit" href="/LogIn" onPress={handleClick}>
-        Log Out
+          LOG OUT
```

Comment:
Is this for log in or log out? href and inner html don't match

Reply:
It is to log out, intentionally. We want logged out users to go straight to the log in page, since if they're logging out, they're clearly done using our site for now and would need to log in again :)